asset-transfer-ledger-queries/chaincode-javascript/asset_transfer_ledger_chaincode.js

```diff
@@ -0,0 +1,473 @@
+/*
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+*/
+
+'use strict';
+const shim = require('fabric-shim');
+const util = require('util');
+
+let Chaincode = class {
+  async Init(stub) {
+    let ret = stub.getFunctionAndParameters();
```

Comment:
We'll want to use the contract API for the new samples.
See https://github.com/hyperledger/fabric-samples/blob/master/asset-transfer-basic/chaincode-javascript/lib/assetTransfer.js as a js reference.